indexer/processor.js

```diff
@@ -780,7 +739,10 @@ class NinaProcessor {
         }
       }
 
-      if (releasePublicKey && blacklist.indexOf(releasePublicKey) === -1) {
+      const restrictedReleases = await axios.get(`${ID_SERVER_ENDPOINT}/restricted`);
+      const restrictedReleasesPublicKeys = restrictedReleases.data.restricted.map(x => x.value);
```

Comment:
this will get called for every tx on indexer startup which means we'll currently hit the id server 46.8k times in a short amount of time - we probably want to figure out a way to limit the amount of calls here

ive made a PR to illustrate one way to bypass this: https://github.com/nina-protocol/nina-indexer/pull/647